tests/test_end_to_end.py

```diff
@@ -1,27 +1,52 @@
 import csv
 import shutil
+from pathlib import Path
 
 import pytest
 
 
+class Study:
+    def __init__(self, study_path):
+        super().__init__()
```

Comment:
nit: I don't think there's any need for `super()` here.

Reply:
No, you're quite right. Boilerplate IDE-generated code. Thanks.